samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,186 @@
+
+## Create an Azure Self-Signed Certificate
+- In Azure portal, create a [Self-Signed Certificate](https://learn.microsoft.com/en-us/azure/key-vault/certificates/quick-create-portal).
+- Create and collect `EncryptionCertificateId`, `Base64EncodedCertificate`, `PRIVATE_KEY_PATH`, and save those value  to update in `.env` file later.
+
+## Run the app (Using Teams Toolkit for Visual Studio Code)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio Code.
+
+1. Ensure you have downloaded and installed [Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
+1. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+1. Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+1. Navigate to the `samples/bot-ai-meeting-helper` folder and open with Visual Studio Code.
+1. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
+1. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+1. In the browser that launches, select the **Add** button to install the app to Teams.
+
+> If you do not have permission to upload custom apps (sideloading), Teams Toolkit will recommend creating and using a Microsoft 365 Developer Program account - a free program to get your own dev environment sandbox that includes Teams.
+
+> When the application is running for the first time, the Teams toolkit will generate an app registration along with a password and other necessary credentials which is specified in "teamsapp.local.yml".
+
+## Setup and use the sample locally 
+1) Navigate to the `samples/bot-ai-meeting-helper/.localConfigs1` and rename to ".localConfigs" and update the values below.
+
+   ```txt
+        BOT_ID="BOT_ID"
+        BOT_PASSWORD="SECRET_BOT_PASSWORD"
+        AZURE_OPENAI_API_KEY="SECRET_AZURE_OPENAI_API_KEY"
+        AZURE_OPENAI_ENDPOINT="AZURE_OPENAI_ENDPOINT"
+        AZURE_OPENAI_DEPLOYMENT_NAME="AZURE_OPENAI_DEPLOYMENT_NAME" 
+        BOT_ENDPOINT="BOT_ENDPOINT"
+        Base64EncodedCertificate="Base_64_Encoded_Certificate"
+        EncryptionCertificateId="Encryption_Certificate_Id"
+        PRIVATE_KEY_PATH="Pem_File_Path"
+        Account_Name="Azure_Storage_Account"
+        Account_Key="Azure_Storage_Account_Key"
+        Table_Name="Azure_Storage_Table"
+        partitionKey="Azure_Storage_Table_PartitionKey"
+        AI_Model="Azure_Open_AI_Model"
+        SubscriptionURL="https://graph.microsoft.com/v1.0/subscriptions"
+        SystemPrompt="Generate a filtered list of action items from meeting transcriptions by user in bullet point user wise categorized with proper format like:  <b> Attendee:</b> 
+        Action Items in bullet points"
+        LocalTimeZone=Asia/Kolkata
+        APPINSIGHTS_INSTRUMENTATIONKEY=""
```

Comment:
Step is missing to create APp insights:
Refer this PR: https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1335/files#diff-820624b3d7f7e4be48d0fc48df9ab543ee07089829a4bf35bd2e808c12637bb9R45

Reply:
Done